packages/eslint-config-node/index.js

```diff
@@ -161,6 +161,7 @@ module.exports = {
         ],
         '@typescript-eslint/no-explicit-any': WARNING,
         '@typescript-eslint/consistent-type-imports': WARNING,
```

Comment:
maybe it would make sense to have

```
parserOptions: {
    project: true
  }

```
next to 

`parser: '@typescript-eslint/parser',`

Reply:
more info: https://typescript-eslint.io/linting/typed-linting/#specifying-tsconfigs

Reply:
this way we would target directly only ts files and we wouldn't need to provide tsconfing path manually